From jacmet@sunsite.dk  Tue Mar 16 13:35:06 2010
From: Peter Korsgaard <jacmet@sunsite.dk>
Date: Fri, 12 Mar 2010 12:33:15 +0100
Subject: USB: gadget/multi: cdc_do_config: remove redundant check
To: m.nazarewicz@samsung.com, gregkh@suse.de, linux-usb@vger.kernel.org
Cc: Peter Korsgaard <jacmet@sunsite.dk>
Message-ID: <1268393595-19744-1-git-send-email-jacmet@sunsite.dk>


cdc_do_config() had a double ret check after fsg_add().

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/multi.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/gadget/multi.c
+++ b/drivers/usb/gadget/multi.c
@@ -211,8 +211,6 @@ static int __init cdc_do_config(struct u
 	ret = fsg_add(c->cdev, c, fsg_common);
 	if (ret < 0)
 		return ret;
-	if (ret < 0)
-		return ret;
 
 	return 0;
 }
